src/video/opengl.h

```diff
@@ -65,6 +65,12 @@ class OpenGLBackend : public ZeroedMemoryAllocator, SpriteEncoder {
 	PaletteID last_sprite_pal = (PaletteID)-1; ///< Last uploaded remap palette.
 	bool clear_cursor_cache = false;           ///< A clear of the cursor cache is pending.
 
+	Point cursor_pos;                    ///< Cursor position
+	bool cursor_in_window;               ///< Cursor inside this window
+	PalSpriteID cursor_sprite_seq[16];   ///< Current image of cursor
+	Point cursor_sprite_pos[16];         ///< Relative position of individual cursor sprites
```

Comment:
Some static_assert or a shared sub-structure would be nice to ensure the length of the arrays here and in _cursor stay in sync.